web/app/themes/ppj/src/js/main.js

```diff
@@ -82,3 +82,14 @@ window.addEventListener('load', function() {
 
 
 }, false);
+
+/**
+ * function to close mobile nav menu if the screen
+ * is resized such that the desktop nav is now visible.
```

Comment:
Something along these lines may be more succinct:

> Close mobile nav if screen becomes >= 1024px wide
> The desktop nav will become visible, so mobile nav must be closed

Reply:
Done.